NFIQ2/NFIQ2Algorithm/src/features/FingerJetFXFeature.cpp

```diff
@@ -110,12 +140,15 @@ FingerJetFXFeature::computeFeatureData(
 	}
 
 	unsigned int minCnt { 0 };
-	if (FRFXLLGetMinutiaInfo(hFeatureSet, &minCnt, nullptr) != FRFXLL_OK) {
+	FRFXLL_RESULT fxResMin = FRFXLLGetMinutiaInfo(
+	    hFeatureSet, &minCnt, nullptr);
+	if (fxResMin != FRFXLL_OK) {
```

Comment:
For this and others, investigate if we should be using `if (FRFXLL_SUCCESS())` as opposed to comparing against `FRFXLL_OK`.

```suggestion
	if (FRFXLL_SUCCESS(fxResMin)) {
```